sdk/test/NetStandard/IntegrationTests/IntegrationTests/General.cs

```diff
@@ -141,7 +144,10 @@ public void TestManualClockCorrection()
         // This test verifies that all service clients are able to
         // correctly handle clock skew errors.
         // By default it only tests a small subset of services.
-        //[Fact]
+        // This is commented out because there is a duplicate version of this
```

Comment:
This comment is still inaccurate, the .NET Framework tests do not run in parallel with the tests in this project.

Reply:
Good catch, will update